students/jungchan/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class UserInformation(models.Model):
+    name        = models.CharField(max_length=20)
+    email       = models.CharField(max_length=30)
+    password    = models.CharField(max_length=20)
+    phonenumber = models.CharField(max_length=20)
```

Comment:
```suggestion
    phone_number = models.CharField(max_length=50)
```
- phonenumber -> phone_number
- max_length가 너무 작은 것 같습니다. 50정도로 늘려주세요.